### ISSUE TITLE
Text and shapes become suddenly blurred

### ISSUE THREAD

**Issue by blueglyph** [jgraph/drawio-desktop#1909]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio-desktop/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Describe the bug**
While using the application, the text of GUI elements (menu, widget, ...) becomes suddenly blurred. The same happens to shapes, either in the left panel or in the document. I couldn't find the pattern that triggers that problem; it's apparently random and may target one or several items, or even the whole window. The blurred items may or may not become sharp again.

**To Reproduce**
- As mentioned before, I couldn't find how to reproduce the problem.

**Expected behavior**
- (rhetorical) I expect widgets, documents, and other items on the GUI to remain sharp and readable.

**Screenshots**
![image](https://github.com/user-attachments/assets/47513a90-10a5-45e1-b13a-15bb55df978c)

**draw.io version (In the Help->About menu of the draw.io editor):**
- draw.io version 24.7.17

**Desktop (please complete the following information):**
- Windows 10 x64

**Additional context**
- GPU: NVIDIA RTX 3070: if you're using the Electron framework, it's a known issue with NVIDIA cards, though I don't know what steps can prevent the problem (likely in the setup - if any).
- A temporary work-around is to disable FXAA in the NVIDIA control panel. It's not desirable, though.
- I found the closed issue #387, according to which a very similar problem had supposedly disappeared in 14.1.8 (or maybe before), though there was no clear conclusion.

**Comment by davidjgraph** [comment]
Reading through, it seems disabling FXAA is the only workaround. But, also various users seem to think we fixed this, #387 , https://www.reddit.com/r/electronjs/comments/1f8r1ja/graphics_issues_with_almost_all_electron_apps/, for example.

Given the volume of desktop users we have, it's in the 10s of millions, it's affecting a very small %, otherwise we'd be overloaded with reports, all electron apps would be. The reddit thread supports that theory. Is a complete hardward list of your system available that you can post?

**Comment by blueglyph** [comment]
> Reading through, it seems disabling FXAA is the only workaround. But, also various users seem to think we fixed this, #387 , https://www.reddit.com/r/electronjs/comments/1f8r1ja/graphics_issues_with_almost_all_electron_apps/, for example.

I mentioned #387 in the issue, but I didn't see various users thinking it's fixed — only one, who didn't seem sure at all. As for the reddit you linked, it's a bit chaotic as usual, but there's no clear consensus, either.
 
> Given the volume of desktop users we have, it's in the 10s of millions, it's affecting a very small %, otherwise we'd be overloaded with reports, all electron apps would be. The reddit thread supports that theory. Is a complete hardward list of your system available that you can post?

Apart from the fact it's an NVIDIA card, there's only one monitor at 1080p, and the scaling in Windows is set to the default 100%. I had the same problem on an older computer with a GTX 980. I can try to resurrect it and do a test on it, and perhaps a laptop too (though I think it's just an Intel chip).

How do you estimate that only a very small percentage is affected (or that there are 10s of millions of users)? I know many people who usually don't bother to report problems, even when they already have a GitHub account; I'm pretty sure many stop using the app or find a work-around. 😅

I can see a lot of discussions about it (I'd already seen quite a number in the past, but I didn't keep track). Here are a few:
- https://github.com/electron/electron/issues/8708 (not sure)
- https://github.com/electron/electron/issues/13646
- https://github.com/electron/electron/issues/18344
- https://github.com/desktop/desktop/issues/17444
- https://github.com/microsoft/vscode/issues/51789
- https://www.reddit.com/r/vscode/comments/9szgkp/visual_studio_code_blurry/
- https://github.com/sedwards2009/extraterm/issues/234
- https://github.com/Nexus-Mods/Vortex/wiki/MODDINGWIKI-Users-FAQ#why-is-vortex-blurry
- https://www.youtube.com/watch?v=mfJ9vfkDeos

There's an FAQ entry about something that looks like it in the Electron project GitHub, but I'm not entirely sure if it's the same problem or a more minor text antialiasing issue:
- https://github.com/electron/electron/blob/main/docs/faq.md#the-font-looks-blurry-what-is-this-and-what-can-i-do 

At some point, it was possible to fix the problem in VSCode by disabling the GPU (see below). That's a work-around that worked for me. I had a glance at the VSCode sources, but it seems this is simply passed on to electron, so perhaps it's not a good solution (perhaps an option in the settings?).
- https://stackoverflow.com/a/60244151/183575
- https://www.reddit.com/r/vscode/comments/9szgkp/comment/e8sk0pj

Bottom line, I'm not sure whether it's something to be fixed on the application side or on the framework side. For example, if Electron uses the GPU, perhaps it's possible to disable FXAA from the app or from Electron.

**Comment by davidjgraph** [comment]
Since this is still the only report, we won't have the resources to follow it up. Apologies and thanks for taking the time to report it.
